src/main/java/edu/harvard/iq/dataverse/ManageGuestbooksPage.java

```diff
@@ -220,7 +220,8 @@ private String getFileName(){
        // The fix below replaces any spaces in the name of the dataverse with underscores;
        // without it, the filename was chopped off (by the browser??), and the user 
        // was getting the file name "Foo", instead of "Foo and Bar in Social Sciences.csv". -- L.A.
-       return  dataverse.getName().replace(' ', '_') + "_GuestbookReponses.csv";
+       // Also removing some chars that have been reported to cause issues with certain browsers
+       return  dataverse.getName().replace(' ', '_').replaceAll("[\\\\/:*?\"<>|,;]", "") + "_GuestbookResponses.csv";
```

Comment:
Since these two regex's are the same, should they be defined once as a constant?

Also, can we have some tests?

Reply:
There in 2 different classes.

Reply:
And neither of these classes have any tests :(

Reply:
moving this method to FileUtil and adding tests

Reply:
So much more readable! Thanks!